openmetadata-ui/src/main/resources/ui/src/pages/DatabaseSchemaPage/SchemaTablesTab.tsx

```diff
@@ -36,21 +34,22 @@ interface SchemaTablesTabProps {
   databaseSchemaDetails: DatabaseSchema;
   tableDataLoading: boolean;
   description: string;
-  entityFieldThreadCount: EntityLinkThreadCount[];
-  editDescriptionPermission: boolean;
-  isEdit: boolean;
-  showDeletedTables: boolean;
+  entityFieldThreadCount?: EntityLinkThreadCount[];
+  editDescriptionPermission?: boolean;
+  isEdit?: boolean;
+  showDeletedTables?: boolean;
```

Comment:
@aniketkatkar97 , why we are making these props optional? if we are doing so then we should have default values or at least it should be guarded.